apps/token-manager/app/src/script.js

```diff
@@ -65,8 +69,16 @@ async function initialize(tokenAddress) {
           default:
             return nextState
         }
+      } else {
+        // Finance event
+        switch (event) {
+          case SYNC_STATUS_SYNCING:
+          case SYNC_STATUS_SYNCED:
+            nextState.syncStatus = event
```

Comment:
Perhaps we could use normal `if` statements for this, and move this to be above the the token event handlers, since they're "special" events?